GOVERNANCE.md

```diff
@@ -1,3 +1,246 @@
+  repositories, resources, and assets.
+* Appointing the members of the Code of Conduct Committee in order to ensure
+  responsiveness, diversity, and good judgement.
+
+## Example decisions
+
+* Decides to expand the project's scope to include new functionality related to data analytics, setting the strategic vision for the next major release.
+* Decides that the project should support an additional Gateway API provider for a specific set of Kuadrant policies
+* Decides to set up a special interest group to investigate and prototype a new type of policy focused on LLM integrations
+
+## Delegated authority
+
+The Steering Committee may choose to delegate its authority to other committees
+as-needed. The committee currently recognizes this delegated authority for:
+
+- n/a
+
+## Committee Meetings
+
+The Steering Committee meets every 2 weeks, or as-needed.
+Meetings are held online, and are public by default.
+
+Given the private nature of some of these discussions (e.g. privacy, private
+emails to the committee, code of conduct violations, escalations, disputes
+between members, security reports, etc.) some meetings are held in private.
+
+Meeting notes are available to members of the
+[kuadrant mailing list](https://groups.google.com/g/kuadrant).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance or community procedures are posted as
+[issues or pull requests in the governance repo](https://github.com/Kuadrant/governance), and
+the Steering Committee invites your feedback there.
+
+## Committee members
+
+Seats on the Steering Committee are held by an individual, not by their
+employer.
+
+The current membership of the committee is currently (listed alphabetically by
+first name):
+
+| &nbsp;                                                         | Member           | Organization |
+| -------------------------------------------------------------- | ---------------- | ------------ |
+
+[TODO: Pending elections]
+
+## Decision process
+
+The Steering Committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing
+written policy, creating, removing, or modifying a committee or SIG, all spending,
+hiring, and contracting, official responses to publicly raised issues, or any
+other decisions that at least two of the members present decide require a vote.
+
+Decisions are made in meetings when a quorum of more than half of the members
+are present (and all members have been informed of the meeting), and may pass
+with more than half the members of the committee supporting it.
+
+## Getting in touch
+
+Open an issue on [the governance repo](https://github.com/Kuadrant/governance) and indicate that you would like
+attention from the steering committee.
+
+## Composition
+
+The steering committee has 3 seats. These seats are
+open to any project contributor. See
```

Comment:
contributor, not maintainers...?! I think the `ROLES.md` starts to make more a little more sense.